CalculateCalendarLogic/CalculateCalendarLogic.swift

```diff
@@ -405,10 +404,10 @@ public struct CalculateCalendarLogic {
      * 指定した年の敬老の日を調べる
      */
     internal func oldPeopleDay(year: Int) -> Int {
-        let cal = Calendar.current as NSCalendar
+        let cal = Calendar(identifier: .gregorian)
         
         func dateFromDay(day: Int) -> NSDate? {
-            return cal.date(era: AD, year: year, month: 9, day: day, hour: 0, minute: 0, second: 0, nanosecond: 0) as NSDate?
+            return cal.date(from: DateComponents(era: AD, year: year, month: 9, day: day, hour: 0, minute: 0, second: 0, nanosecond: 0)) as NSDate?
```

Comment:
__[SwiftLint]__

Line should be 120 characters or less: currently 147 characters


<!-- Comment by [Sider](https://sider.review) -->